universe/Planet.h

```diff
@@ -1,6 +1,7 @@
 #ifndef _Planet_h_
 #define _Planet_h_
 
+#include "Enums.h"
```

Comment:
I will roll back the 3 lines that use PT_TERRAN and PT_SIZE, and then I won't need to move Enums.h

This was left over from an earlier attempt at this PR, when tried being more aggressive in using default member inits (to try to get rid of as many init lists as possible).